### PR TITLE
Eating is now possible on bedrock

### PR DESCRIPTION
**Pull request by francois76**
Bedrock players are now able to eat (previously, saturation and health was not updated server side).

### CODE REVIEW

**Comment by HoverEpic** [comment]
Thank you, I'll check that tomorrow.

**Comment by shaokeyibb** [comment]
Wow！Good fix！

File: proxy/src/main/java/org/dragonet/proxy/network/translator/pe/PEInventoryTransactionPacketTranslator.java [hunk 1]
```diff
@@ -1,181 +0,0 @@
-package org.dragonet.proxy.network.translator.pe;
```
Comment (HoverEpic) [inline proxy/src/main/java/org/dragonet/proxy/network/translator/pe/PEInventoryTransactionPacketTranslator.java]: How do you manage to delete a whole file and commit that? Next time, please check all your changes bebore commiting.

**Review by HoverEpic — COMMENTED** [review]
I think you should redo this pull request with only necessary changes.

**Comment by lukeeey** [comment]
Nah just squash and merge

**Comment by francois76** [comment]
I'm a beginner with git, so I had modified a file by mistake and I didn't wanted it to be commited, so instead of deleting its commit, I have commited its deletion :(
I've squashed my commit and I will do another pull request with one commit ;)

**Review by HoverEpic — APPROVED** [review]
Beautifull!

**Comment by DefinitlyEvil** [comment]
cool!

**Pull request merged**